api/test/wfl/integration/executor_test.clj

```diff
@@ -28,12 +28,22 @@
     fixtures/temporary-postgresql-database))
 
 (deftest test-validate-terra-executor-with-valid-executor-request
-  (is (executor/terra-executor-validate-request-or-throw
-       {:name                       "Terra"
-        :workspace                  testing-workspace
-        :methodConfiguration        testing-method-configuration
-        :methodConfigurationVersion testing-method-configuration-version
-        :fromSource                 "importSnapshot"})))
```

Comment:
You could close up the map in the following let now that `:methodConfigurationVersion` is gone.

Reply:
Thanks!  Will do this throughout.